### ISSUE TITLE
Use PythonImportRequirement

### ISSUE THREAD

**Issue by jayvdb** [coala/coala-bears#1655]

**Comment by arjunsinghy96** [comment]
To be used in place of PipRequirement ??

**Comment by jayvdb** [comment]
yup.  only for bears which import directly into the bear.  i.e. we want to remove the imports.

This is highly desirable before I can finish https://github.com/coala/coala-bears/issues/1000 .
